src/mintpy/prep_aria.py

```diff
@@ -417,14 +418,301 @@ def write_ifgram_stack(outfile, unwStack, cohStack, connCompStack, ampStack=None
         for dsName in ['unwrapPhase','coherence','connectComponent']:
             f[dsName].attrs['MODIFICATION_TIME'] = str(time.time())
 
-    print(f'finished writing to HD5 file: {outfile}')
+    print(f'finished writing to HD5 file: {outfile}\n')
     dsUnw = None
     dsCoh = None
     dsComp = None
     dsAmp = None
     return outfile
 
 
+# OPTIONAL - ARIA corrections troposphereTotal, ionosphere, solidearthtides
+def write_iono_stack(outfile, ionStack, box=None, 
```

Comment:
This function is almost the same as the `write_ifgram_stack()`, could you re-use it instead of creating a new one?